OpenRA.Mods.Common/Traits/HitShape.cs

```diff
@@ -19,6 +19,12 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("Shape of actor for damage calculations.")]
 	public class HitShapeInfo : ConditionalTraitInfo
```

Comment:
You `self.Trait<BodyOrientation>()`, so this needs a `Requires<BodyOrientationInfo>`.